kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/GenEditingDetailsController.java

```diff
@@ -557,6 +554,24 @@ private void populateSemanticDetails() {
         }
     }
 
+    private void updateUIStamp(StampEntityVersion stampEntityVersion){
+
+        State state = stampEntityVersion.state();
+        stampViewControl.setStatus(state == null? "" : getViewProperties().calculator().getDescriptionTextOrNid(state.nid()));
+
+        // -- Time
+        updateTimeText(stampEntityVersion.time());
+
+        EntityFacade author = stampEntityVersion.author();
+        stampViewControl.setAuthor(author == null? "" : getViewProperties().calculator().getDescriptionTextOrNid(author.nid()));
+
+        EntityFacade module = stampEntityVersion.module();
+        stampViewControl.setModule(getViewProperties().calculator().getDescriptionTextOrNid(module.nid()));
+        EntityFacade path = stampEntityVersion.path();
+        stampViewControl.setPath(getViewProperties().calculator().getDescriptionTextOrNid(path.nid()));
+
+    }
```

Comment:
we should use the updateUIStamp method that already exists that takes a StampViewModel instance.